yarn-in-docker.sh

```diff
@@ -5,4 +5,4 @@
 # $yarn-in-docker.sh workspace antora-ui-camel run build
 
 docker build -t camel-website .
-docker run --rm -it -v $(pwd):/work:Z --workdir /work camel-website yarn $*
+docker run --rm -it --env-file yarn-in-docker.env -v $(pwd):/work:Z --workdir /work camel-website yarn $*
```

Comment:
```suggestion
docker run --rm -it -e GITHUB_TOKEN=${GITHUB_TOKEN} -e CAMEL_ENV=${CAMEL_ENV} -v $(pwd):/work:Z --workdir /work camel-website yarn $*
```

Reply:
It is not really a flexible approach if for some reason we need more env variables, we will then have to modify the script too moreover you still need to set the env variables locally

Reply:
However if you prefer your approach, I can change it. Let me know WYT

Reply:
I'm okay either way. My motivation was simplicity. We most likely won't be adding new environment variables and we'll remove the support for `HUGO_PARAMS_GitHub*` environment variables in the future, so having two `-e` parameters and having those be inherited from the shell seemed simpler to me.

Reply:
As you wish